samples/ipc_call_interface/consumer/wrappers/math_wrapper.h

```diff
@@ -49,19 +49,34 @@ typedef int64_t subtract_model_out;
 /*
  * math class constructor.
  */
-az_ulib_result math_create(math_handle handle);
+inline az_ulib_result math_create(math_handle handle) {
```

Comment:
I'm curious if this is going to be using the `az_core`, these should probably be `AZ_INLINE` in the long term [(what that expands to here)](https://github.com/Azure/azure-sdk-for-c/blob/191b654cbda636167d6e85b2abd2b2cd6b36e5b7/sdk/inc/azure/core/_az_cfg.h#L60-L66). For now, I would do
```suggestion
static inline az_ulib_result math_create(math_handle handle) {
```

Reply:
here and elsewhere

Reply:
Yes, and probably the NO_DISCARD as well.

Reply:
Should probably implement corresponding unit tests if you're going to use AZ_NODISCARD. Maybe in separate PR. Fine with everything else as long as the pipeline succeeds after adding static specifiers.

Reply:
AZ_NODISCARD is part of the Azure Core, they already test it. This is a sample, so, no test on it.

---
In reply to: [564147176](https://github.com/Azure/azure-ulib-c/pull/56#discussion_r564147176) [](ancestors = 564147176)